ci/before_deploy.sh

```diff
@@ -10,4 +10,4 @@ gcloud auth activate-service-account ${CI_SERVICE_ACCOUNT} --key-file travis-ser
 gcloud config set compute/zone europe-west1-c
 gcloud config set core/project ${PROJECT_ID}
 gcloud container clusters get-credentials api-platform-demo --zone europe-west1-c --project ${PROJECT_ID}
-helm init --upgrade
+helm init --force-upgrade
```

Comment:
Why?

Reply:
Because tiller server version is newer than client one.